data/json/items/melee.json

```diff
@@ -3425,6 +3503,32 @@
         "category" : "weapons"
     },
     {
+    "id": "zweihander_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "zweihänder",
+    "description": "This is a huge two-handed sword from Germany.  You're not quite sure it's supposed to be that bendy.",
+    "rarity": 30,
```

Comment:
Probably an old value for old drop system.
Should be removed as it doesn't do anything nowadays.